pudb/py3compat.py

```diff
@@ -2,27 +2,20 @@
 import sys
```

Comment:
If we're getting rid of all this we can just delete this file, and the `__future__` imports everywhere. It's better to just use Python 3 names rather than replacing them with the old Python 2 names.

Reply:
Fair enough. 7f1b08a